tests/versions/ruby.yml

```diff
@@ -3,5 +3,5 @@ RUBY_AGENT:
   # release;latest -> gem elastic-apm
   # release;version -> gem elastic-apm, version
   - 'github;master'
-  - 'github;2.x'
```

Comment:
NIT: only changes the order of the versions to follow the same rules for all APM Agents.